packages/web/src/common/store/cache/users/sagas.ts

```diff
@@ -67,14 +68,21 @@ function* retrieveUserByHandle(handle: string, retry: boolean) {
     handle = handle[0]
   }
 
-  const { data: users = [] } = yield* call(
-    [sdk.full.users, sdk.full.users.getUserByHandle],
-    {
-      handle,
-      userId: OptionalId.parse(userId)
+  try {
+    const { data: users = [] } = yield* call(
+      [sdk.full.users, sdk.full.users.getUserByHandle],
+      {
+        handle,
+        userId: OptionalId.parse(userId)
+      }
+    )
+    return userMetadataListFromSDK(users)
+  } catch (e) {
+    if (((e as Error).cause as Response)?.status === 404) {
```

Comment:
Kinda feels like this should let the error bubble up to the caller and be handled there?

Reply:
can handle level up, that's good, but isResponseError isn't sufficient because it only checks if it's a response error, not the type of error (404 status)

Reply:
You use it to get the type hint so you can do `error.response.status`

Reply:
👍